src/apps/benchmarks/benchmarkH3UniEdge.c

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2020 Uber Technologies, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "benchmark.h"
+#include "geoCoord.h"
+#include "h3api.h"
+
+// Fixtures
+H3Index edges[6] = {0};
+H3Index hex = 0x89283080ddbffff;
```

Comment:
Is there a particular significance to this exactly hex ID, or was it chosen arbitrarily? It would be useful to have a comment on this in the code either way.

Reply:
It's arbitrary, copy/pasta from another benchmark. I can add a comment to both.

Reply:
Appears to be Pier 39: https://observablehq.com/@nrabinowitz/h3-index-inspector#89283080ddbffff

Reply:
Interesting. But good to know there was no explicit significance to it.